content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,321 @@
+
+   If the email advertises a price reduction, it makes sense to target leads with the lost reason:
+   *too expensive*. To do so, add rules that exclude all reasons except *too expensive*.
+
+   .. image:: lost_leads_email/too-expensive.png
+      :align: center
+      :alt: A list of filter rules that exclude all lost reasons except for Too Expensive.
+
+.. _analyze_results:
+
+Analyze the results
+===================
+
+After sending a lost leads reactivation email, marketing teams can use the smart buttons along the
+top of the email to analyze the results and determine follow-up actions.
+
+Clicking on any of the smart buttons opens a list of the records matching that button's specific
+criteria.
+
+.. image:: lost_leads_email/smart-buttons.png
+   :align: center
+   :alt: The Mailing page of a sent email showing the smart buttons along the top of the page.
+
+The smart buttons include:
+
+- :guilabel:`Sent`: the total number of emails that were sent.
+- :guilabel:`Opened`: the percentage of recipients that opened the email.
+- :guilabel:`Replied`: the percentage of recipients that replied to the email.
+- :guilabel:`Clicked`: the click-through rate (%) of recipients that clicked on a link in the email.
+- :guilabel:`Leads`: the number of leads that have been created in the *CRM* pipeline as a result of
+  the email campaign.
+- :guilabel:`Quotations`: the number of quotations that have been created in the *Sales* application
+  as a result of the email.
+- :guilabel:`Invoiced`: the total revenues generated as a result of the email campaign, via invoices
+  sent to and paid by customers. These values are recorded in either the *Invoicing* or *Accounting*
+  application, depending on whichever is installed on the database.
+- :guilabel:`Received`: the percentage of recipients that received the email.
+- :guilabel:`Bounced`: the percentage of emails that bounced :dfn:`not delievered`
+- :guilabel:`Ignored`: the number of recipients that received the email but have not interacted with
+  it in a meaningful way (i.e. opened, clicked)
+
+Email nurturing
+---------------
+
+*Email nurturing* (sometimes referred to as *lead nurturing*) is the process of sending a series of
+timely and relevant *nudge* emails to connect with a lead, build a deeper relationship, and
+ultimately convert the lead into a sale.
+
+The point of nurturing is to keep the email campaign "visible" or at the top of a lead's inbox until
+they are ready to buy. There are many approaches to effective lead nurturing, but it often involves:
+
+- sending an initial email (such as a lost leads reactivation email)
+- sending a follow-up email each week (or according to specific triggers) for the duration of the
+  campaign
+- continuously analyzing the results to see what approaches have resulted in sales
+- continuously adjusting the approach to remain "visible" at the top of the lead's inbox and get a
+  meaningful response from the lead
```

Comment:
Capitals for the first letter and punctuation at the end?

Reply:
Sure, that seems to be the team's preference anw 👍